README.md

```diff
@@ -41,28 +41,60 @@ Depending on target operating system, _INAV Configurator_ is distributed as _sta
 ### Linux
 
 1. Visit [release page](https://github.com/iNavFlight/inav-configurator/releases)
-1. Download Configurator for Linux platform (linux32 and linux64 are present)
-1. Extract tar.gz archive
-1. Make the following files executable:
-   * inav-configurator `chmod +x inav-configurator`
-   * (5.0.0+) chrome_crashpad_handler `chmod +x chrome_crashpad_handler`
-1. Run INAV Configurator app from unpacked folder
+2. Download Configurator for Linux platform (linux32 and linux64 are present)
+   *  **.rpm** is the Fedora installation file. Just download and install using `sudo dnf localinstall /path/to/INAV-Configurator_linuxNN_x.y.z..rpm` or open it with a package manager (e.g. via Files)
+   *  **.deb** is the Debian/Ubuntu installation file. Just download and install using `sudo apt install /path/to/INAV-Configurator_linuxNN_x.y.z.deb` or open it with a package manager (e.g. via the File Manager)
+   *  **.tar.gz** is a universal archive. Download and continue with these instructions to install
+3. Change to the directory containing the downloaded **tar.gz** file
+4. download [this](https://raw.githubusercontent.com/iNavFlight/inav-configurator/master/assets/linux/inav-configurator.desktop) file to the same directory. Its filename should be `inav-configurator.desktop`.
+5. Extract **tar.gz** archive 
+```
+tar -C /tmp/ -xf INAV-Configurator_linuxNN_x.y.z.tar.gz
+```
+   **NN** is the bits of your OS. **x.y.z** is the INAV Configurator version number.
+
+6. If this is the first time installing INAV Configurator, create a home for its files
+```
+sudo mkdir /opt/inav
+sudo chown $USER /opt/inav
+```
+7. Move the temporary files in to their home 
+```
+mv /tmp/INAV\ Configurator /opt/inav/inav-configurator
+```
+8. Update the application icon.
+```
+sudo mkdir /opt/inav/inav-configurator/icon
+sudo cp /opt/inav/inav-configurator/images/inav_icon_128.png /opt/inav/inav-configurator/icon
+```
+9. As a one off, move the desktop file into the applications directory 
+```
+sudo mv inav-configurator.desktop /usr/share/applications/
+```
+10. Make the following files executable:
+   * inav-configurator `chmod +x /opt/inav/inav-configurator/inav-configurator`
+   * (5.0.0+) chrome_crashpad_handler `chmod +x /opt/inav/inav-configurator/chrome_crashpad_handler`
+11. Run INAV Configurator app from unpacked folder
```

Comment:
From a beginner point of view, should we be obvious here and list the file path `/opt/inav/inav-configurator/inav-configurator`
or how to find it in applications?

Reply:
Finding it in "Applications" depends on the user's selected DE. It will be different in GNOME, KDE, Xfce and the myriad of others.

Reply:
I usually would run things from the terminal anyway.

It is probably a good idea to make it obvious that you should run `/opt/inav/inav-configurator/inav-configurator` to launch inav as the readon why they did the chmod +x on the steop before may not be obvious to a beginner.